src/cephadm/cephadm

```diff
@@ -3348,13 +3348,13 @@ def install_base_units(ctx, fsid):
     compress
     sharedscripts
     postrotate
-        killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x 'ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror' || true
+        chown 167:167 /var/log/ceph/%s/*.log; killall -q -1 ceph-mon ceph-mgr ceph-mds ceph-osd ceph-fuse radosgw rbd-mirror cephfs-mirror || pkill -1 -x 'ceph-mon|ceph-mgr|ceph-mds|ceph-osd|ceph-fuse|radosgw|rbd-mirror|cephfs-mirror' || true
```

Comment:
you think we can avoid hardcoding 167 here`

Reply:
are you thinking we just add them as constants at the top of the file?

Reply:
is used in some places:

https://github.com/ceph/ceph/blob/3e23850bafa4ac48bb2f2f657fca19541540a511/src/cephadm/cephadm#L2788

Reply:
nice, thanks @sebastian-philipp! I updated it to use this